Cargo.toml

```
@@ -10,24 +10,24 @@ edition = "2021"
 repository = "https://github.com/stackabletech/opa-operator"
 
 [workspace.dependencies]
-axum = "0.6"
-built = { version = "0.6", features = ["chrono", "git2"] }
+axum = "0.7"
+built = { version ="0.7", features = ["chrono", "git2"] }
 clap = "4.3"
```

Clap should not be breaking
```suggestion
built = { version ="0.7", features = ["chrono", "git2"] }
clap = "4.5"
```